litmus-portal/graphql-server/server.go

```diff
@@ -59,10 +59,11 @@ func main() {
 		AllowedOrigins:   []string{"*"},
 		AllowCredentials: true,
 	}).Handler)
-
+	go synchandler.RecurringHubSync() //go routine for syncing hubs for all users
```

Comment:
Not sure this should be put it here because if there is no hub connected with the portal.

Reply:
for go-routine, i tested it with no hubs it is working fine, it will call the go-routine but won't sync anything as the hubs array will be empty.